.github/labeler.yml

```diff
@@ -0,0 +1,102 @@
+arch:aarch32:
+  - compiler/arm/**/*
+  - port/linuxarm/**/*
+
+arch:aarch64:
+  - compiler/aarch64/**/*
+  - port/linuxaarch64/**/*
+
+arch:power:
+  - compiler/p/**/*
+  - fvtest/compilerunittest/p/**/*
+  - jitbuilder/p/**/*
+  - port/aix*/**/*
+  - port/linuxppc*/**/*
+
+arch:riscv:
+  - compiler/riscv/**/*
+  - port/linuxriscv64/**/*
+
+arch:x86:
+  - compiler/x/**/*
+  - jitbuilder/x/**/*
+  - port/linux386/**/*
+  - port/linuxamd64/**/*
+  - port/osx/**/*
+  - port/win*/**/*
+  - thread/osx/**/*
+  - thread/win32/**/*
+
+arch:z:
+  - compiler/z/**/*
+  - jitbuilder/z/**/*
+  - port/linuxs390*/**/*
+  - port/zos390/**/*
+  - thread/zos/**/*
+
+build / configure:
+  - buildenv/**/*
+  - cmake/**/*
+
+ci:
+  - buildenv/jenkins/**/*
+  - scripts/Appveyor-ProcessTests.ps1
+  - scripts/build-on-travis.sh
+  - ./.appveyor.yml
+  - ./.travis.yml
+  - ./azure-pipelines*
+
+cmake:
+  - cmake/**/*
+
+comp:compiler:
+  - compiler/**/*
+
+comp:diagnostic:
+  - ddr/**/*
+
+comp:gc:
+  - gc/**/*
+
+comp:jitbuilder:
+  - jitbuilder/**/*
+
+comp:port:
+  - port/**/*
+
+comp:test:
+  - fvtest/**/*
+
+comp:thread:
+  - thread/**/*
+
+comp:tril:
+  - fvtest/compilertriltest/**/*
+  - fvtest/tril/**/*
+
+documentation:
+  - doc/**/*
+  - ./**/*.md
+
+os:aix:
+  - port/aix*/**/*
+  - thread/aix*/**/*
+
+os:linux:
+  - port/linux*/**/*
+  - thread/linux*/**/*
+
+os:macos:
+  - port/osx/**/*
+  - thread/osx*/**/*
```

Comment:
Minor nit:  Keep it `osx` to be consistent with `port` and the fact that there aren't any specializations of osx.

Reply:
Fixed in [force-push](https://github.com/eclipse/omr/compare/02fed63e6206221d73bd82917a10bc0ac274c3f6..94f6fb6ea30c69ca960f027c5b75e14f2cf63186).